config/test/svg.transform.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
```

Comment:
This is heavily inspired by web app's version...  ok, ok.... it's a blatant copy-paste. 😄